packages/create-catalyst/src/commands/integration.ts

```diff
@@ -0,0 +1,118 @@
+import { Command } from '@commander-js/extra-typings';
+import { exec as execCb } from 'child_process';
+import { outputFileSync, writeJsonSync } from 'fs-extra/esm';
+import * as semver from 'semver';
+import { promisify } from 'util';
+import * as z from 'zod';
+
+const exec = promisify(execCb);
+
+interface Manifest {
+  name: string;
+  dependencies: string[];
+  devDependencies: string[];
+  environmentVariables: string[];
+}
+
+export const integration = new Command('integration')
+  .requiredOption('--integration-name <name>', 'Formatted name of the integration')
```

Comment:
Could we have this be an argument instead of an option? e.g. `pnpm create @bigcommerce/catalyst@latest integration example-integration`.

🍹 Would also love some validation and/or have the name always be lower-kebab-case (recommended through validation instead of implicitly doing it).